lib/hanami/events/adapter/memory.rb

```diff
@@ -7,40 +7,16 @@ class Memory
         def initialize(logger: nil, **)
```

Comment:
And also, I think we should rename this adapter to `MemorySync` because `Memory` doesn't obvious. WDYT?